digitaltwins/samples/service/javascript/get_digital_twin.js

```diff
@@ -15,10 +15,6 @@ const { inspect } = require('util');
 async function main() {
   const deviceId = process.env.IOTHUB_DEVICE_ID;
 
-  // Create digital twin service client
-  const credentials = new IoTHubTokenCredentials(process.env.IOTHUB_CONNECTION_STRING);
-  const digitalTwinServiceClient = new DigitalTwinServiceClient(credentials);
```

Comment:
I think you need to authenticate with something.  And that is the hub connection string.